tests/opentelemetry/test_logs.py

```diff
@@ -504,3 +504,62 @@ def test_otel_trace_log_correlation():
     assert (
         int(attributes["span_id"], 16) == span_context.span_id
     ), f"Expected span_id_hex to be set to {attributes['span_id']} but found: {span_context.span_id}"
+
+
+@pytest.mark.skipif(
+    EXPORTER_VERSION < MINIMUM_SUPPORTED_VERSION,
+    reason=f"OpenTelemetry exporter version {MINIMUM_SUPPORTED_VERSION} is required to export logs",
+)
+@pytest.mark.snapshot()
+@pytest.mark.subprocess(ddtrace_run=True, env={"DD_LOGS_OTEL_ENABLED": "true"})
```

Comment:
Since the tests are testing DD_LOGS_OTEL_ENABLED:true, does that mean users who have this setting would notice the impact? Asking because I wonder if the release notes need to call out DD_LOGS_OTEL_ENABLED specifically and whether the changelog comment in this readthedocs page should be adjusted: https://ddtrace.readthedocs.io/en/stable/configuration.html?highlight=dd_logs_otel_enabled#DD_LOGS_OTEL_ENABLED